data/error_mail_template_data.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
the parent folder is already named "data", so wouldn't it be enough to just call this file **error_mail_template.xml** or this is naming convention coming from odoo docs ?